components/MobileNavigation.js

```diff
@@ -20,23 +20,53 @@ export default class MobileNavigation extends Component {
 
   state = {
     showMobileNav: false,
-    expandMobileNav: false
+    expandMobileNav: false,
+    nowSelectMenu: null,
+    scrollPosition: null
   }
 
   componentDidMount () {
+    const { navs } = this.props
     window.addEventListener('scroll', this.handleScroll)
+    const scrollPosition = navs.reduce((obj, nav, index) => {
+      const navScroll = {
+        name: nav.label,
```

Comment:
Don’t depend on the label since it is for displaying purpose only.

Instead, use `nav.href` here, since it is used to ‘ref’er to sections on the page.